src/main/resources/metadata/AEM-19.json

```
@@ -0,0 +1,5 @@
+{
+  "remediation": {
+    "constantCost": "5min"
```

```suggestion
    "constantCost": "30min"
```

I'd argue here, as it's not only about choosing the right strategy, the code change is small. However, one need to understand problem and choose the right one.